PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/README.md

```diff
@@ -107,4 +107,16 @@ integration, telemetry connectivity, and Azure Toolkit integration.
   Credential (Entra ID Authentication). If not, then use Azure Key Credential or Connection Strings based
   authentication. Please refer to
   the [Azure SDK for Java documentation](https://learn.microsoft.com/en-us/java/api/com.azure.identity.defaultazurecredential?view=azure-java-stable)
+  for additional information.
+
+7. #### Kusto Queries Having a Time Interval in the Query String
+
+- **Anti-pattern**: Writing KQL queries with hard-coded time intervals directly in the query string.
+- **Issue**: This approach makes queries less flexible and harder to troubleshoot.
+  -**Recommendation**: Consider using the `QueryTimeInterval` parameter in the client method parameters to specify the
+  time interval for the query.
+  By passing the time range as an argument in the method call, you make it easier to troubleshoot and understand the
+  context of an API call.
+  Please refer to
+  the [Azure SDK for Java documentation](https://learn.microsoft.com/en-us/java/api/com.azure.monitor.query.models.querytimeinterval?view=azure-java-stable)
```

Comment:
```suggestion
  the [Azure SDK for Java documentation](https://learn.microsoft.com/java/api/com.azure.monitor.query.models.querytimeinterval?view=azure-java-stable)
```

Just a nit, should remove all language locales from URLs so that the browser can select the most appropriate choice for the user.

Reply:
thanks:) I took the en-us out from all links too in my tooltip popup on hover changes:)